scripts/restartdaq

```diff
@@ -6,13 +6,9 @@ cat << EOF
 usage: $0 options
 
 OPTIONS:
--w sort windows after start
--d dss-test setup
--p select partition (same as used last)
--s silent (do not email jana)
--c enable core files
--C (LCLS2 DAQMGR Hutches ONLY!) Select a cnf to use.
--u UED .cnf file
```

Comment:
Was nobody using the options we're dropping here?

Reply:
They were either not used or have been rewritten to also allow for a longer command line option as I moved away from getopts.